selsort.py

```diff
@@ -0,0 +1,22 @@
+def selectionsort( aList ):
+  for i in range( len( aList ) ):
+    least = i
+    for k in range( i + 1 , len( aList ) ):
+      if aList[k] < aList[least]:
+        least = k
+
+    swap( aList, least, i )
+
+
+def swap( A, x, y ):
+  tmp = A[x]
+  A[x] = A[y]
+  A[y] = tmp
+l=[]
+size=int(input("enter size of array: "))
+for i in range(size):
+	x=int(input("enter element: "))
```

Comment:
Line contains following spacing inconsistencies:
- Tabs used instead of spaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpl3r34pcv/selsort.py
+++ b/tmp/tmpl3r34pcv/selsort.py
@@ -15,7 +15,7 @@
 l=[]
 size=int(input("enter size of array: "))
 for i in range(size):
-	x=int(input("enter element: "))
+    x=int(input("enter element: "))
 	l.append(x)
 
 selectionsort(l)
```